st_ui_builder/lib/ui_builder/type_def.dart

```diff
@@ -40,3 +39,52 @@ typedef GroupComparatorCallback = int Function(
   GroupHeaderData,
   GroupHeaderData,
 );
+
+abstract class StKey {
+  const StKey(this._value);
+
+  final String _value;
+
+  String get value => _value;
+
+  @override
```

Comment:
@override is unnecessary here
and (I believe) if you want these methods to actually work, rather than requiring @override in subclasses, you can implement them in an EXTENSION on the abstract base class, and you then don't need to implement in your subclasses